src/app/shared/model/feeder-podcast.model.ts

```diff
@@ -120,6 +134,12 @@ export class FeederPodcastModel extends BaseModel {
     } else {
       data.itunesCategories = [];
     }
+
+    data.explicit = this.explicit || null;
```

Comment:
I only moved this line of code to alphabetize the attributes being set (yes, I am crazy).
In `encode`, it seems to be these get set to `null`, and in `decode` to empty string `''`, which makes sense to me as we don't want to save empty string values in the API when it should be a `null`.
@cavis can you confirm if `null` or empty string `''` is appropriate here?